### DISCUSSION TITLE
Compatibility between zigbee2mqtt and HA-Add-On Silicon Labs Multiprotocol

### DISCUSSION THREAD

**Discussion by GerdRuetten** [Koenkk/zigbee2mqtt#20859] [General]
Hello community,

I would like to use the ZBDongle-E with multiprotocol in HomeAssistant via the “Silicon Labs Multiprotocol” add-on.

In the current add-on version it says in the chnagelog

"Zigbee2MQTT does not yet support this version of the Gecko SDK. Do not update if you are using Z2M!"

Is there any (time-)plan to update z2m to ensure compatibility?

Thank you in advance!

Best regards
Gerd

**Comment by Koenkk** [comment]
Already supported in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) and in the 1 February release.

**Comment by GerdRuetten** [comment]
> Already supported in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) and in the 1 February release.

great!! 

Thank you Koen .. you make a perfect job
